platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,135 @@
+# Proposal Title Goes Here
+
+---
+
+## Summary
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+| Metadata           | Entities                            | 
```

Comment:
Once an proposal is merged we can not see in the file if the proposal is already implemented or not. we should add a column for that